store/views.py

```diff
@@ -77,6 +97,44 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data={
+        'message':None,
+    }
+    book_id=request.POST['bid']
+    book=BookCopy.objects.get(pk=book_id)
+    try:
+        book.borrower=None
+        book.borrow_date=None
+        book.status=True
+        book.save()
+        msg="success"
+    except:
+        msg='failure'
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.